slack_done.gs

```diff
@@ -133,14 +120,24 @@ function done_process_modal(userid,view){
   
   // update log sheet
   var log_sheet = new LogSheetWrapper();
-  log_sheet.appendRow([new Date(), uniqueid, 'admin','confirmDoneUser', return_message]);
+  log_sheet.appendRow([new Date(), args.uniqueid, 'admin','confirmDoneUser', return_message]);
 }
 
 
 
-function done(uniqueid, channelid, userid, requestNextStatus, completionLastDetails){
+function done(args){
   ///// COMMAND: /DONE
-    
+  
+  var uniqueid = args.uniqueid;
```

Comment:
No need to make any changes now, but I was skimming the v8 runtime docs and extracting arguments like this can be done cleanly using argument destructing assignment.

https://developers.google.com/apps-script/guides/v8-runtime#destructuring_assignments